supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-enterprise-linux.adoc

```diff
@@ -46,6 +610,193 @@ For documentation questions, contact rhel-docs@redhat.com.
+In a Red Hat Identity Management environment, POSIX attributes for groups include:
+
+  * `cn`, the group's name
+  * `gidNumber`, the group number (GID)
+
+These attributes identify users and groups as separate entities.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[refs]]
+==== image:images/yes.png[yes] refs (noun)
+*Description*: Represents a branch in OSTree. Refs always resolve to the latest commit. For example, `rhel/8/x86_64/edge`.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:ostree[ostree]
+
+[discrete]
+[[remote]]
+==== image:images/yes.png[yes] remote (noun)
+*Description*: The HTTP or HTTPS endpoint that hosts the OSTree content. This is analogous to the baseurl for a `yum` or `dnf` repository.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:ostree[ostree]
+
+[discrete]
+[[replication-agreement]]
+==== image:images/yes.png[yes] replication agreement (noun)
+*Description*: A replication agreement is an agreement between two IdM servers in the same IdM deployment. The replication agreement ensures that the data and configuration is continuously replicated between the two servers.
+IdM uses two types of replication agreements: _domain replication_ agreements, which replicate identity information, and _certificate replication_ agreements, which replicate certificate information.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:idm-deployment[IdM deployment]
+
+[discrete]
+[[revision]]
+==== image:images/yes.png[yes] revision (noun)
+*Description*: Revision (Rev) represents SHA-256 for a specific OSTree commit.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:ostree[ostree]
+
+[discrete]
+[[rpm-ostree]]
+==== image:images/yes.png[yes] rpm-ostree (noun)
+*Description*: A hybrid image or system package that hosts operating system updates.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:ostree[ostree]
+
+[discrete]
+[[smart-card]]
+==== image:images/yes.png[yes] smart card (noun)
+*Description*: A smart card is a removable device or card used to control access to a resource. They can be plastic credit card-sized cards with an embedded integrated circuit (IC) chip, small USB devices such as a Yubikey, or other similar devices. Smart cards can provide authentication by allowing users to connect a smart card to a host computer, and software on that host computer interacts with key material stored on the smart card to authenticate the user.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[sssd]]
+==== image:images/yes.png[yes] SSSD (noun)
+*Description*: The System Security Services Daemon (SSSD) is a system service that manages user authentication and user authorization on a RHEL host. SSSD optionally keeps a cache of user identities and credentials retrieved from remote providers for offline authentication.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[sssd-backend]]
+==== image:images/yes.png[yes] SSSD backend (noun)
```

Comment:
Just wanted to check that "backend" one word is correct in this instance. Normally per ISG rules, "back end" as a noun is two words

Reply:
I checked it with the author, and it is okay for him to change it according to the ISG.